### PR TITLE
rest: offer overall user quota consumption

### LINKED ISSUE #289: rest: combine usage and limits of different quota resources into dict
We should combine quotas into some kind of dict (maybe something similar like [here](https://github.com/reanahub/reana-client/pull/425/files#diff-4f27f08c3f46f0b755c1d5ac1a1e2dcaR72)) and group the usage and limits of different quota resources before returning the response. 

_Originally posted by @audrium in https://github.com/reanahub/reana-server/pull/288#discussion_r498788299_

### PR DESCRIPTION
**Pull request by diegodelemos**
* Closes #289.

### CODE REVIEW

File: docs/openapi.json [hunk 1]
```diff
@@ -647,17 +647,33 @@
...
                   "access_token": "Drmhze6EPcv0fN_81Bj-nA",
-                  "cpu_usage": "70536",
-                  "disk_usage": "408",
                   "email": "user@reana.info",
-                  "id": "00000000-0000-0000-0000-000000000000"
+                  "id": "00000000-0000-0000-0000-000000000000",
+                  "quota": {
+                    "cpu": {
+                      "limit": "200000",
+                      "usage": "70536"
```
Comment (mvidalgarcia) [inline docs/openapi.json:655]: Shouldn't it be an int if it's type "number"?

File: reana_server/rest/users.py [hunk 2]
```diff
@@ -148,98 +176,6 @@ def get_user():  # noqa
         return jsonify({"message": str(e)}), 500
 
 
-@blueprint.route("/users", methods=["POST"])
-def create_user():  # noqa
```
Comment (mvidalgarcia) [inline reana_server/rest/users.py:152]: Any specific reason why we get rid of this? Maybe we can specify it in the commit message.

File: docs/openapi.json (hunk 1 above)
Comment (diegodelemos) [reply docs/openapi.json:655]: That's what I thought, but then I realised we won't be able to fit big integers into integers... so we are left with two options: `string` or `number` without format (representing "any number" see [here](https://swagger.io/docs/specification/data-models/data-types/#numbers)).

File: reana_server/rest/users.py (hunk 2 above)
Comment (diegodelemos) [reply reana_server/rest/users.py:152]: Mainly because we do not use it. I will specify in the commit message, thanks for pointing out :)

**Review by mvidalgarcia — APPROVED** [review]

**Pull request merged**